### DISCUSSION TITLE
Configuration help

### DISCUSSION THREAD

**Discussion by landocarlesien** [damianeickhoff/HaCasa#31] [General]
Hi All - Need some help. I followed the download and configuration steps. Didn't work. Do I need to import the custom cards first for it to work? When I add the required info to configuration.yaml, it auto generates the HaCasa dashboard and wont let me edit.

**Comment by frischform** [comment]
You need to edit it here:

/config/dashboard/HaCasa/views

every view have his configuration .yaml file
